### PR TITLE
Add build and run instructions to README

### PR DESCRIPTION
**Pull request by annkamsk**
I'm probably going to reference this repo in the Trusted Types related PRs for external projects. It'll be good to have at least short instructions for build and run here.

### CODE REVIEW

**Comment by engelsdamien** [comment]
@koto for visibility

File: README.md [hunk 1]
```diff
@@ -11,3 +11,30 @@ is potentially incompatible with Trusted Types, tsec emits compilation errors.
...
+## Build and run
+
+### Build
+```
+  # Clone the repository
+  git clone https://github.com/googleinterns/tsec
+  cd tsec
+  
+  # Install gulp
+  npm install gulp
```
Comment (Siegrift) [inline README.md:24]: Why do we need gulp? It works for me even if I skip this step. If we need it it should probably be in `devDependencies`

File: README.md [hunk 2]
```diff
@@ -11,3 +11,30 @@ is potentially incompatible with Trusted Types, tsec emits compilation errors.
...
+  
+```
+
+### Run
+
+To run a check for the compatibility with Trusted Types in the project containing `tsconfig.json` use:
+```
+  {PATH_TO_TSEC}/bin/tsec -p tsconfig.json
+```
+Add `--no-Emit` flag to skip the output from compilation.
```
Comment (Siegrift) [inline README.md:40]: What are we skipping here? If I run `{PATH_TO_TSEC}/bin/tsec -p tsconfig.json` it works the same with or without a flag for me.

Comment (uraj) [reply README.md:40]: Hmm, the flag should be `--noEmit`

File: README.md (hunk 1 above)
Comment (uraj) [reply README.md:24]: We don't need gulp. I guess Anna added this because she works on VSCode which requires gulp

**Pull request merged**